src/libraries/System.Text.Json/src/System/Text/Json/JsonHelpers.Date.cs

```diff
@@ -117,12 +117,6 @@ public static bool IsValidDateTimeOffsetParseLength(int length)
             return IsInRangeInclusive(length, JsonConstants.MinimumDateTimeParseLength, JsonConstants.MaximumEscapedDateTimeOffsetParseLength);
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool IsValidDateTimeOffsetParseLength(long length)
```

Comment:
Why delete this helper?

Reply:
It wasn't being used anymore so I cleaned it up. The code that was calling it (in Utf8JsonReader.TryGet) is now calling `IsInRangeInclusive` directly so it can pass in the max length based on encoded string or not. You want me to put it back with the other feedback items for a follow-up?